assignments/arrays.js

```diff
@@ -63,34 +63,69 @@ let inventory = [{"id":1,"car_make":"Lincoln","car_model":"Navigator","car_year"
 
 // ==== Challenge 1 ====
 // The dealer can't recall the information for a car with an id of 33 on his lot. Help the dealer find out which car has an id of 33 by logging the car's year, make, and model in the console log provided to you below:
-console.log(`Car 33 is a *car year goes here* *car make goes here* *car model goes here*` );
+
+console.log(inventory[32]);
 
 
 
 // ==== Challenge 2 ====
 // The dealer needs the information on the last car in their inventory.  What is the make and model of the last car in the inventory?  Log the make and model into the console.
-let lastCar = 0;
-console.log();
+let lastCar = (inventory.length)-1;
+console.log(inventory[lastCar]);
```

Comment:
Have to make sure we are correctly referencing the right place in the array. In line 73 you are setting lastCar equal to a number as inventory.length is going to return the number of items in the array. In order to access the last item we need to say ```inventory[inventory.length - 1]```. This will reference the last object in the inventory array. Once we assign the value of ```inventory[inventory.length -1]``` to ```lastCar``` we can reference it by simply calling lastCar. No need to continue using the bracket notation.

```
let lastCar = inventory[inventory.length - 1];
console.log(lastCar);
```